### ISSUE TITLE
Dash is not one of top ten coins in the CMC list and it is a problem for attracting investors

### ISSUE THREAD

**Issue by awesomedash** [DashNetwork/dn-issues#3]
This is a chicken and egg problem. Unfortunately the legitimacy of coins is somewhat perceived based on their position in the CMC list, though we know that claimed MCs may not necessarily have anything to do with the network size and usage. Currently the fact that Dash is not even in the top 30 and constantly going lower is a huge barrier in attracting new investors.

**Comment by awesomedash** [comment]
This topic came up in the Dash discord #mno [here](https://discord.com/channels/484546513507188745/523262303433523200/807008787176620033).

**Comment by Wikiwako** [comment]
IMHO being in the Top Ten on MC should never be a priority of DCG or any other team. We should focus on being the first project to be used globally for common payments, which is the reason most of us were attracted to Bitcoin/Crypto in the first place. With time price will be a function of usability and not popularity. 

It is a fact that raising awareness is difficult without the popularity, but it is also true that Dash Evolution has been in the works for more than 5 years and people get tired of waiting. We should all be patient. The strategy is ok. 

"Timing, perseverance, and ten years of trying will eventually make you look like an overnight success."

**Comment by awesomedash** [comment]
I agree on what's important is usability and I think being one of top 10 coins increases adoption, which helps usability because then it is much better supported by merchants, exchanges and other major player in the market. This also brings more funds to the network. My concern is that if we lose too much time other projects, who are in a much better financial position, will copy Dash features or new projects replace it. Also growing Dash network wouldn't be possible without having significantly larger group of investors.
